modules/rules_erlang/3.2.0/MODULE.bazel

```diff
@@ -24,3 +24,7 @@ use_repo(
     "getopt_src",
     "xref_runner_src",
 )
+
+register_toolchains(
+    ":erlang_toolchain_external",
```

Comment:
sorry for the churn -- you need to write "//:erlang_toolchain_external" here (with the //)